frontend/src/pages/warehouse/ModalEditWarehouse.js

```diff
@@ -137,49 +137,55 @@ function ModalEditWarehouse(props) {
 
     const editWarehouseHandler = (e) => {
         e.preventDefault();
-        let location = `${warehouseDto.addressDto.state.state} 
+
+        let warehouseUpdateDto = {};
+        let dispatchersId = dispatchers.map(dispatcher => dispatcher.id);
+        warehouseUpdateDto = {
+            ...warehouseDto,
+            dispatchersId: dispatchersId,
+            customerId: props.currentCustomerId,
+            irrelevantDispatchersId: dispatcherDeleteList
```

Comment:
if this list is derived from some other list in the state or any other place, we could have a function defined in the component, which give use this list whenever we want. We don't need to keep 2 lists in the state, if one of them can be derived from another one.